src/services/swap_quote_service.ts

```diff
@@ -0,0 +1,19 @@
+import { Orderbook, SupportedProvider } from '@0x/asset-swapper';
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
+
+import { SWAP_PATH } from '../constants';
+import { errorHandler } from '../middleware/error_handling';
+import { createSwapQuoteRouter } from '../routers/swap_quote_router';
+
+// tslint:disable-next-line:no-unnecessary-class
+export class SwapQuoteHttpService {
```

Comment:
We might just want to call this `SwapHttpService` since it will also power endpoints like `/swap/tokens` down the road

Reply:
We can do this refactor when we add the endpoints. YAGNI till that spec is fully complete

Reply:
I think it makes sense to rename. It is just the path not the subpaths usually that we name this after.